File: CPP/Program/PetShop.cpp
#pragma once
#include <bits/stdc++.h>
using namespace std;

class PetShop{
    // Attributes
    private:
        string id;
        string nama_produk;
        string stok_produk;
        int harga_produk;
    //constructors
    public:
    // Default constructor
        PetShop(){
            this->id = "";
            this->nama_produk = "";
            this->stok_produk = "";
            this->harga_produk = 0;
        }

        PetShop(string id, string nama_produk, string stok_produk, int harga_produk){
            this->id = id;
            this->nama_produk = nama_produk;
            this->stok_produk = stok_produk;
            this->harga_produk = harga_produk;
        }

        // Getter and Setter
        string getId(){
            return this->id;
        }

        string getNama_produk(){
            return this->nama_produk;
        }

        string getStok_produk(){
            return this->stok_produk;
        }

        int getHarga_produk(){
            return this->harga_produk;
        }

        void setId(string id){
            this->id = id;
        }

        void setNama_produk(string nama_produk){
            this->nama_produk = nama_produk;
        }

        void setStok_produk(string stok_produk){
            this->stok_produk = stok_produk;
        }

        void setHarga_produk(int harga_produk){
            this->harga_produk = harga_produk;
        }

        //destructor
        ~PetShop(){
        }

};

File: Python/Program/PetShop.py
class PetShop:
    # Constructor, no variable di awal nyobain
    def __init__(self, id="", nama_produk="", stok_produk="", harga_produk=0):
        self.id = id
        self.nama_produk = nama_produk
        self.stok_produk = stok_produk
        self.harga_produk = harga_produk
    
    # Setters
    def set_id(self, id):
        self.id = id
    
    def set_nama_produk(self, nama_produk):
        self.nama_produk = nama_produk
    
    def set_stok_produk(self, stok_produk):
        self.stok_produk = stok_produk
    
    def set_harga_produk(self, harga_produk):
        self.harga_produk = harga_produk
    
    # Getters
    def get_id(self):
        return self.id
    
    def get_nama_produk(self):
        return self.nama_produk
    
    def get_stok_produk(self):
        return self.stok_produk
    
    def get_harga_produk(self):
        return self.harga_produk
